tests/python_on_whales/components/test_container.py

```diff
@@ -232,6 +232,22 @@ def test_run_with_preserve_fds(podman_client: DockerClient):
     ):
         assert os.read(read_fd, 7) == b"foobar\n"
 
+def test_run_with_timezone(podman_client: DockerClient):
+    output = podman_client.container.run(
+        "ubuntu", ["date", "+'%Z'"], tz="local", remove=True
+    )
+    # Check the timezone in the container matches that in the host.
+    assert output == datetime.now().astimezone().strftime("'%Z'")
+
+def test_create_start_with_timezone(podman_client: DockerClient):
+    with podman_client.container.create(
+        "ubuntu", ["sleep", "infinity"], tz="GMT", stop_timeout=1
```

Comment:
I'm not sure why you have `stop_timeout=1` here - if you actually wanted to remove the timeout you'd want `stop_timeout=0`, but I wouldn't have thought it'd be needed with `sleep` which should get stopped immediately by `SIGTERM`?

Reply:
was copied from another test case I don't think it is needed.